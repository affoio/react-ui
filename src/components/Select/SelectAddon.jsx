import styled from 'styled-components';
import ComponentAddon from '../ComponentAddon/ComponentAddon';

const SelectAddon = styled(ComponentAddon)`
  width: 24px;
  height: 24px;
`;

export default SelectAddon;
