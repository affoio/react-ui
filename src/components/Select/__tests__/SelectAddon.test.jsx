import React from 'react';
import renderer from 'react-test-renderer';
import SelectAddon from '../SelectAddon';

describe('<SelectAddon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SelectAddon />);

    expect(tree).toMatchSnapshot();
  });
});
