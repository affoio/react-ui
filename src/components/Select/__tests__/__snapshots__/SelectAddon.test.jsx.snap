// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SelectAddon renders correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  margin-right: 4px;
  width: 24px;
  height: 24px;
}

<div
  className="c0"
/>
`;
